accessibility-checker-engine/help-v4/en-US/input_autocomplete_valid.html

```diff
@@ -81,7 +83,7 @@ <h3 id="ruleMessage"></h3>
 </form>
 ```
 
-Example code includes material copied from or derived from [Using HTML5 Autocomplete attributes, technique H98](https://www.w3.org/WAI/WCAG22/Techniques/html/H98). Copyright © 2017-2018 W3C® (MIT, ERCIM, Keio, Beihang)
+Example code includes material copied from or derived from [H98: Using HTML 5.2 autocomplete attributes](https://www.w3.org/WAI/WCAG22/Techniques/html/H98). Copyright © 2017-2018 W3C® (MIT, ERCIM, Keio, Beijing )
```

Comment:
Beihang was correct.
```suggestion
Example code includes material copied from or derived from [H98: Using HTML 5.2 autocomplete attributes](https://www.w3.org/WAI/WCAG22/Techniques/html/H98). Copyright © 2017-2018 W3C® (MIT, ERCIM, Keio, Beihang)
```